membership-common/src/main/scala/com/gu/zuora/api/PaymentGateway.scala

```diff
@@ -23,6 +23,14 @@ case object StripeAUPaymentIntentsMembershipGateway extends PaymentGateway {
   val gatewayName = "Stripe PaymentIntents GNM Membership AUS"
   override val forCountry = Some(Country.Australia)
 }
+case object StripeUSMembershipGateway extends PaymentGateway {
+  val gatewayName = "Stripe Gateway GNM Membership USA"
+  override val forCountry = Some(Country.US)
+}
+case object StripeUSPaymentIntentsMembershipGateway extends PaymentGateway {
+  val gatewayName = "Stripe PaymentIntents GNM Membership USA"
+  override val forCountry = Some(Country.US)
+}
```

Comment:
why do we need both, I assumed that everything new was payment intents nowadays?

Reply:
@johnduffell

Thanks for this. I have copied what we have done for the AUS Stripe Gateway, but if you confirm that we don't need that anymore, I am happy to delete the US one that I have created and the AUS that was already there?

Reply:
Reference to summary chat after a call between us and Rupert: https://chat.google.com/room/AAAA6HaLTYQ/HD_8XQ2HBK4/FS2h9zgg6_s?cls=10